recipe/scripts/deactivate-win.sh

```diff
@@ -1,3 +1,4 @@
+export PATH=$(echo $PATH | sed "s?$JAVA_HOME/bin:??")
```

Comment:
I have done this in another recipe on Linux. Not sure this will work with posix cli on windows.

Reply:
Rather than mucking about with the PATH I recommend putting a link in the standard places, e.g., as done in https://github.com/conda-forge/openjdk-feedstock/blob/c205caaa29d765fcf79dd4ea6bcd9f162879ee53/recipe/build.sh#L221

Reply:
I don't know much about windows. The `bin` directory contains both `.exe` and `.dll`.
Can I create link only for `.exe` files?
My understanding from https://github.com/conda-forge/openjdk-feedstock/issues/86 is that we don't want the `.dll` (like `zip.dll`) in the default bin directory to avoid conflicts.

I tried using the following command but it didn't work:

```
for %%f in (%INSTALL_DIR%\bin\*.exe) do (
    mklink "%LIBRARY_BIN%\%%~nf.exe" %%f
)
```

Looks like you have to be admin on windows to create links. There is `mklink /J` but that is for directory only.

Do you have another solution than changing the PATH?

Reply:
I think we want to link .exe files only, and link them to `%PREFIX%\bin\`

Reply:
How do you create link on windows as normal user?

Reply:
Oh hmmm, one more reason to hate windows :-(

Reply:
Again, not a windows expert but I've never seen a recipe creating links under windows.
So I think the current implementation (modifying the PATH) is probably the way to go. I tested the `activate.bat` and `deactivate.bat`. They work as expected. I'm just not 100% sure about the `activate-win.sh` / `deactivate-win.sh` as I don't know when they are used exactly. But on Linux, this shell script would work fine.

Reply:
I have a Windows VM and tested it locally.